app/controllers/bulkrax/exporters_controller.rb

```diff
@@ -0,0 +1,149 @@
+require_dependency "bulkrax/application_controller"
+
+module Bulkrax
+  class ExportersController < ApplicationController
+    include Hyrax::ThemedLayoutController
+
+    before_action :set_exporter, only: [:show, :edit, :update, :destroy]
+    with_themed_layout 'dashboard'
+
+    # GET /exporters
+    def index
+      add_breadcrumb t(:'hyrax.controls.home'), main_app.root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb 'Exporters', bulkrax.exporters_path
+      @exporters = Exporter.all
```

Comment:
can we break paginating this and the importer in to its own ticket? this would be a good one for @jpengst or @bkiahstroud to pick up.